contrib/pai_vscode/package.nls.json

```diff
@@ -17,8 +17,9 @@
     "paiext.cluster.dashboard.open": "Open Dashboard",
     "paiext.cluster.job.list": "Open Job List Externally",
     "paiext.cluster.job.submit": "Submit Job to PAI Cluster",
-    "paiext.cluster.job.create-config": "Create PAI Job Config JSON",
-    "paiext.cluster.job.create-yaml-config": "Create PAI Job Config YAML",
+    "paiext.cluster.job.create-config": "Create PAI Job Config",
+    "paiext.cluster.job.create-config-v1": "Create PAI Job Config JSON",
```

Comment:
Please use `v1`/`v2` instead of `JSON`/`YAML`

Reply:
Done.